dbms/src/Server/Server.cpp

```diff
@@ -1049,6 +1050,25 @@ int Server::main(const std::vector<std::string> & /*args*/)
         LOG_FMT_INFO(log, "tiflash proxy thread is joined");
     });
 
+    /// get server info.
+    {
+        auto * request = new diagnosticspb::ServerInfoRequest();
```

Comment:
1. `request` need to be `delete` when it never be used.
2. Alloc `request` in stack seems a better way, like `diagnosticspb::ServerInfoRequest request;`.

Reply:
fixed!